README.md

```diff
@@ -1,11 +1,11 @@
 # city-explorer-api
 
 **Author**: Jason
-**Version**: 1.0.0 (increment the patch/fix version number if you make more commits past your first submission)
+**Version**: 1.2.1 (increment the patch/fix version number if you make more commits past your first submission)
 
 ## Overview
 
-This is my first time creating and using a server. The app we are creating takes in a location and presents a map and weather data about the location.
+This is my first time creating and using a server. The app we are creating takes in a location and presents a map and weather data about the location. The app also presents national park information for nearby parks.
```

Comment:
Great job keeping this up to date with information, overview, and other related notes.